app/lib/actions/katello/content_view/force_destroy.rb

```diff
@@ -0,0 +1,21 @@
+module Actions
+  module Katello
+    module ContentView
+      class ForceDestroy < Actions::EntryAction
+        def plan(content_view, options)
+          options[:check_ready_to_destroy] = false
+          options[:skip_environment_check] = true
+          options[:force_destory_cv] = true
+          sequence do
+            plan_action(Katello::ContentView::Remove, content_view, options) unless (options[:content_view_versions].empty? && options[:content_view_environments].empty?)
+            plan_action(Katello::ContentView::Destroy, content_view, options)
```

Comment:
Second check options[:content_view_environments].empty? is probably redundant cause empty version would mean empty content_view_environments..